pytorch_blade/tests/disc/pdl/pdll_files/cpu/dequant_gemm_quant.pdll

```diff
@@ -70,7 +72,7 @@ Pattern TorchDequantLinearQuantOp {
     let infos = CreateTorchCustomCall(attr<"\"op\"">, inputs, outputs);
 
     /// 2. set attrs that are used by bladedisc.
-    SetAttr(infos.op, attr<"\"call_target_name\"">, attr<"\"ral_pdll_qgemm\"">);
+    SetAttr(infos.op, attr<"\"call_target_name\"">, attr<"\"ral_pdll_qgemm_s8s8s8f32_pc\"">);
     SetAttr(infos.op, attr<"\"input_placements\"">, attr<"\"h,h\"">);
```

Comment:
the number of items not equals to the number inputs.

Reply:
done